app/controllers/manual_sections_controller.rb

```diff
@@ -17,10 +17,7 @@ def new
   end
 
   def create
-    section_data = params["section"]
-    section_data["manual_content_id"] = params["manual_content_id"]
-
-    @section = Section.new(section_data)
+    @section = Section.new(section_params.merge("manual_content_id" => params["manual_content_id"]))
```

Comment:
Worth noting that it's only relevant in this case because `permit` (the source of `section_params`) returns a `HashWithIndifferentAccess`. If it was a normal `Hash`, the suggested version wouldn't be equivalent.